auditlog/models/rule.py

```diff
@@ -294,9 +284,13 @@ def create_logs(self, uid, res_model, res_ids, method,
         log_model = self.env['auditlog.log']
         for res_id in res_ids:
             model_model = self.env[res_model]
-            res_name = model_model.browse(res_id).name_get()
+            # Avoid recursivity with the 'read' method called by 'name_get()'
+            res_name = "%s,%s" % (res_model, res_id)
+            if method is not 'read':
+                name = model_model.browse(res_id).name_get()
+                res_name = name and name[0] and name[0][1] or res_name
```

Comment:
I am not very comfortable with this piece of code. We need to qualify the log with a good name, but a call to `name_get` + `read` log enabled  = falling in recursivity.